gameswap-service/pom.xml

```diff
@@ -45,12 +45,17 @@
         <dependency>
             <groupId>io.dropwizard</groupId>
             <artifactId>dropwizard-auth</artifactId>
-            <version>0.9.2</version>
+            <version>${dropwizard.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
fixing the databind version because a dependency I'm about to add pulls in an older version